lib/landing_session.js

```diff
@@ -84,6 +86,19 @@ class LandingSession extends Session {
         process.exit(1);
       }
     }
+
+    // Check for and maybe assign any unmarked deprecations in the codebase.
+    const unmarkedDeprecations = await getUnmarkedDeprecations();
+    const unmarkedDepCount = unmarkedDeprecations.length;
+    if (unmarkedDepCount > 0) {
+      cli.startSpinner('Assigning deprecation numbers to DEPOXXX items');
+      await updateDeprecations(unmarkedDeprecations);
+
+      // Amend the last commit with the updated deprecation items.
+      await runAsync('git', ['commit', '--amend', '--no-edit']);
```

Comment:
do we need to `git add` the modified files?

Reply:
heh yes - fixed 🤦‍♀️